### PR TITLE
Items Guide

### LINKED ISSUE #32: Developer Guides: Adding items

### PR DESCRIPTION
**Pull request by dicedpixels**
Covers the items basics from the wiki:

- [Adding an Item](https://fabricmc.net/wiki/tutorial:items)
- [Item Groups](https://fabricmc.net/wiki/tutorial:itemgroup)
- [Adding a tooltip](https://fabricmc.net/wiki/tutorial:tooltip)

What's left:

- [Crafting Recipes](https://fabricmc.net/wiki/tutorial:recipes)
- [Adding Armor](https://fabricmc.net/wiki/tutorial:armor)
- [Adding Tools](https://fabricmc.net/wiki/tutorial:tools)
- ~[Making a Custom Shield in Minecraft ](https://fabricmc.net/wiki/tutorial:shield)~ I don't think a tutorial based on a 3rd party library should be here
- [Adding Enchantments](https://fabricmc.net/wiki/tutorial:enchantments)
- [Model Predicate Providers](https://fabricmc.net/wiki/tutorial:model_predicate_providers)

### CODE REVIEW

File: develop/items/items.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+@[code lang=java transcludeWith=:::1](@/reference/latest/src/main/java/com/example/docs/item/FabricDocsReferenceItems.java)
+
+You can then use the vanilla registry system to register the item. Items are registered into `Registries.ITEM`.
+
+@[code lang=java transcludeWith=:::2](@/reference/latest/src/main/java/com/example/docs/item/FabricDocsReferenceItems.java)
+
+You can simplify the item creation and registration as follows:
+
+```java
```
Comment (MattiDragon) [inline develop/items/items.md:37]: There's a random `s` here

Comment (MattiDragon) [reply develop/items/items.md:37]: It's one line below, mobile ui fail

File: reference/latest/src/main/resources/assets/fabric-docs-reference/lang/en_us.json [hunk 2]
```diff
@@ -1,4 +1,8 @@
 {
   "effect.fabric-docs-reference.tater": "Tater",
-  "item.minecraft.potion.effect.tater": "Tater Potion"
+  "item.minecraft.potion.effect.tater": "Tater Potion",
```
Comment (RooftopJoe) [inline reference/latest/src/main/resources/assets/fabric-docs-reference/lang/en_us.json:3]: Is it really best practice to name the key `item.minecraft.potion.effect.tater`? I've always figured that the `minecraft` bit is supposed to be reserved for vanilla, kind of like a namespace.

Comment (dicedpixels) [reply reference/latest/src/main/resources/assets/fabric-docs-reference/lang/en_us.json:3]: Yes, but by design potion effect names go under the `minecraft` namespace.

**Comment by IMB11** [comment]
Closing in favour of #116 Will merge the tooltip guide into that PR.

**Pull request closed without merging**